### DISCUSSION TITLE
Q: Does auto-paste work on the iPad with dual view (one for viewing the PDF, one for pasting the notes)?

### DISCUSSION THREAD

**Discussion by ekliao** [RyotaUshio/obsidian-pdf-plus#161] [Q&A]
Hello, I just started using PDF++. Love its great auto-paste functionality on the desktop. I wonder if this feature works on the iPad as well? Has anyone used auto-paste successfully on iPad? Thank you.

**Comment by RyotaUshio** [comment]
Hi @ekliao, this plugin had some problems on mobile, but I believe they have been already fixed by the recent updates. Let me know if it's still not working on the latest version of PDF++.
